generate-release-notes.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+set -eo pipefail
+
+usage() {
+    echo "Usage:"
+    echo "  $0 AL2_GPU_NVIDIA_VERSION AL2_GPU_CUDA_VERSION [AL1_CONTAINERD_VERSION]"
+    echo "  AL1_CONTAINERD_VERSION should not be provided if there is no AL1 AMI release"
+    echo "Example:"
+    echo "  $0 470.182.03 11.4 1.4.13 # AL1 release included"
+    echo "  $0 470.182.03 11.4        # No AL1 release"
+}
+
+# Parameters
+al2_gpu_nvidia_version=$1
```

Comment:
Another thought on these parameters - would you prefer current implementation, or prefer we add those params to https://github.com/aws/amazon-ecs-ami/blob/main/generate-release-vars.sh#L74 to fully automate this shell execution?

Reply:
These parameters are not known until we build the AMIs which happens after running `generate-release-vars.sh` script. So, that's why the parameters exist in `generate-release-notes.sh` instead.